extraConfigDpu.py

```diff
@@ -296,10 +296,6 @@ def helper(h: host.Host, node: NodeConfig) -> Optional[host.Result]:
                 logger.error_and_exit(f"Failed to bring up IPU net device on {h.hostname()}")
             ret = h.run(f"test -d /sys/class/net/{cfg.dpu_net_interface}")
 
-        # Temporarily we will need to manually create the vfs on each DPU host
-        # TODO: Remove when no longer required
-        h.run_or_die(f"echo {SRIOV_NUM_VFS} | sudo tee /sys/class/net/{cfg.dpu_net_interface}/device/sriov_numvfs")
-
```

Comment:
At what point do the vfs get created by the daemon?

Out of curiosity, what would happen if we encounter the bad state this part of the code is trying to handle (i.e. the node does not have the net devs for the IPU, cold boot is required to bring them up).

Reply:
Created when the host.yaml is applied.

Shouldn't matter since host.yaml is not applied. So even if it is in a bad state nothing will happen.